.gitignore

```diff
@@ -0,0 +1,71 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
```

Comment:
I don't understand why it's still pushing those in addition.
you have any idea?